control/trajectory_follower_nodes/test/trajectory_follower_test_utils.hpp

```diff
@@ -69,6 +69,15 @@ inline geometry_msgs::msg::TransformStamped getDummyTransform()
   transform_stamped.child_frame_id = "base_link";
   return transform_stamped;
 }
+// TODO remove this to modify latlon muxer publish topic only when data is ready
```

Comment:
[NITS] I think it is better to use `TODO(name):` style, because it follows the Google coding style.
https://google.github.io/styleguide/cppguide.html#TODO_Comments

Reply:
0f962f9 
cc : @TakaHoribe